pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -88,6 +88,15 @@ autoPatchelf() {
         --extra-args "${patchelfFlagsArray[@]}"
 }
 
+autoPatchelfEachOutput() {
```

Comment:
Naming nit: based on the comments below, this ought to be named `autoPatchelfPostFixup`